pkg/nsx/services/securitypolicy/firewall.go

```diff
@@ -585,17 +586,19 @@ func (service *SecurityPolicyService) createOrUpdateT1SecurityPolicy(obj *v1alph
 func (service *SecurityPolicyService) createOrUpdateVPCSecurityPolicy(obj *v1alpha1.SecurityPolicy, createdFor string) error {
 	var err error
 	var finalGetNSXSecurityPolicy *model.SecurityPolicy
-
 	isDefaultProject := false
+
 	vpcInfo, err := service.getVPCInfo(obj.ObjectMeta.Namespace)
 	if err != nil {
 		return err
 	}
-	if vpcInfo.ProjectID == common.DefaultProject {
-		isDefaultProject = true
+	isDefaultProject, err = service.vpcService.IsDefaultNSXProject(vpcInfo.OrgID, vpcInfo.ProjectID)
```

Comment:
Remove `isDefaultProject := false` and change `isDefaultProject, err = ...` to `isDefaultProject, err := ...`

Reply:
Thanks, fixed.